docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -30,32 +30,41 @@ Replaced with:
 apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 ----
+The following documentation is intended to assist administrators in converting exising **ClusterLogging.logging.openshift.io** and **ClusterLogForwarder.logging.openshift.io** specifications to the new observability API.  We've provided an overview of the changes, as well as complete **ClusterLogForwarder** resource examples for several common use cases.
 
-NOTE: There is no automated upgrade provided by the *cluster-logging-operator*
+No Automated Update::
+Given the numerous combinations in which log collection, forwarding, and storage can be configured, there is no automated upgrade provided by the *Cluster Logging Operator*. The existing v5.9 operator must be uninstalled before the new v6.0 operator can be installed.
```

Comment:
@cahartma I don't know if we want to change this but per our conversation yesterday this is not correct.  One can change the channel and the operator will upgrade